### DISCUSSION TITLE
Opera webläsare

### DISCUSSION THREAD

**Discussion by Manhem** [stefansundin/privatkopiera#215] [General]
Finns det en möjlighet att man kan få en plugin även till Opera??

**Comment by stefansundin** [comment]
Jag använder inte Opera själv men jag tror att man kan installera Privatkopiera ändå på Chrome Web Store, eftersom Opera är numera baserad på Chrome.

Skriv gärna om du får det att fungera, och om det är några problem i funktionen. Jag kan tänka mig att installera Opera för att fixa småproblem.

Vet inte om jag vill publicera Privatkopiera i Operas egna store. Detsamma gäller för Edge. Men jag skapade precis en poll för att försöka se hur många användare som använder andra webbläsare. Rösta här: https://github.com/stefansundin/privatkopiera/discussions/216

**Comment by Manhem** [comment]
Tack för svaret.
Jag provar och återkommer med ett svar senare.
